src/operator/mxnet_op.h

```diff
@@ -32,6 +35,13 @@ int get_num_threads(const int N);
       i < (n); \
       i += blockDim.x * gridDim.x)
 
+inline cudaDeviceProp cuda_get_device_prop() {
```

Comment:
If the function is only used for cuda, guard the function definition with `#ifdef __CUDACC__`.